setup.py

```diff
@@ -17,8 +17,6 @@
 SETUP_REQUIREMENTS = ["setuptools>=28", "setuptools_scm"]
 REQUIREMENTS = [
     "ecl",
-    "equinor-libres",
```

Comment:
Not sure if it is correct to remove this. I did a quick search for `libres` / `res` but could not find any direct imports. Is it used directly?

Reply:
It probably is an indirect dependency, originally added because both ert and libres was explicitly compiled from source prior to pypi.